README.md

```diff
@@ -47,19 +60,42 @@ val v1CartCreatedPersister = persister[CartCreatedV1]("cart-created")
 
 // spray-json persister for V2 but with support for migration
 // of data writen in the V1 format.
-val v2CartCreatedPersister = persister[CartCreatedV2, V2]("cart-created",
-  from[V1].to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
+val v2CartCreatedPersister = persister[CartCreatedV2, V2](
+  "cart-created",
+  from[V1]
+    .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
 )
 
 // spray-json persister for V3 but with support for migration
 // of data writen in the V1 and V2 formats.
-val v3CartCreatedPersister = persister[CartCreatedV3, V3]("cart-created",
+val v3CartCreatedPersister = persister[CartCreatedV3, V3](
+  "cart-created",
   from[V1]
     .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
     .to[V3](_.update('timestamp ! set[Long](System.currentTimeMillis)))
 )
 ```
 
+To use persisters, they need to be encapsulated in a `StaminaAkkaSerializer` that is registered, in the 
+application.conf, for the event classes that need to be persisted. Here the provided `Persistable` marker trait is used
+and the serializer is registered for `Persistable` classes.
+
+```scala
+class CartCreatedV3(...) extends Persistable
+class CartSerializer extends StaminaAkkaSerializer(v3CartCreatedPersister, ...)
```

Comment:
Maybe this application is only about carts ;) This is just an example to get people going. I was really missing. It is no worse than the example above about different versions. Maybe it is a good idea to add some comments?
I could not think of a better name, without having a really general name like `ApplicationSerializer`. 
Your point is more about taste/style/preferences. Stamina is not the resource to say what style you should or should not keep to.